internal/consensus/state_test.go

```diff
@@ -114,7 +118,7 @@ func TestStateProposerSelection2(t *testing.T) {
 
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
-	cs1, vss := makeState(ctx, t, config, logger, 4) // test needs more work for more than 3 validators
+	cs1, vss := makeState(ctx, t, config, logger, makeStateArgs{}) // test needs more work for more than 3 validators
```

Comment:
Maybe convert this TODO to an issue?

Reply:
Perhaps, although truthfully I'm not certain I understand what the comment is calling for :thinking:

Reply:
Yeah, I wasn't sure either. 👀